src/stories/1.index.stories.jsx

```diff
@@ -21,10 +21,99 @@ import {
 import Checkmark from '../components/Widgets/AnimatedIcons/Checkmark'
 import Cross from '../components/Widgets/AnimatedIcons/AnimatedCross'
 
+import AddressSelect from '../components/Widgets/Form/AddressSelect'
+
+ // wallet dummy data
+const dummyWallets =
+  {
+  '0x2685F863Ddb456601783A57A1C3E9F8f3ebc6c3B': {
+    balance: '3',
+    name: 'Account 1',
+    addressType: 'wallet',
+  },
+  '0xabc5F863Ddb456601783A57A1C3E9F8f3ebc6c3B': {
+    balance: '0',
+    name: 'Account 2',
+    addressType: 'wallet',
+  },
+  '0xdef5F863Ddb456601783A57A1C3E9F8f3ebc6c3B': {
+    balance: 0,
+    name: 'Account 3',
+    addressType: 'wallet',
+  },
+   '0xghi5F863Ddb456601783A57A1C3E9F8f3ebc6c3B': {
+    balance: 0.0,
+    name: 'Account 4',
+    addressType: 'wallet',
+  },
+  '0xzzz5F863Ddb456601783A57A1C3E9F8f3ebc6c3B': {
+    balance: 0.01,
```

Comment:
nit: some values as numbers, some as strings. Please have all be strings.

Reply:
yes, and you can try small values like ".000001" ether or even in hexadecimal format  to verify it renders correctly. when passed into BN like `new BN(wallet.balance).toString()` usually normalizes the output pretty well

Reply:
Done.